csatar-plugins/csatar/models/association/fields.yaml

```diff
@@ -4,6 +4,10 @@ fields:
         span: auto
         required: 1
         type: text
+    name_abbreviation:
```

Comment:
Shouldn't this be added to the columns.yaml as well? Until now that contained all the columns.